apps/dialtone-documentation/docs/guides/getting-started/index.md

```diff
@@ -177,7 +177,7 @@ cd ./path/to/dialtone
 Dialtone uses [Gulp](https://gulpjs.com/) to automate its various workflows. Run the following command to install Gulp and all other project dependencies:
 
 ```bash
-    npm install
+npm install
```

Comment:
I think this should be `pnpm install`, right?

Reply:
And below, in the Building Dialtone section, the command should be `nx start:dialtone`

Reply:
Thanks! fixed